.github/workflows/deployment-ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - name: Build with Gradle
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: --stacktrace --info build -x apiCheck
```

Comment:
This will cause the apiCheck to happen on snapshots which we don''t want

Reply:
I know, and I wanted to change this.

We would now always have to update the api dump when we change api, which will tie the diff in .api files with the actual change to the same commit. This will always make us aware that we changed something and will prevent unwanted breaking changes by mistake.

This is also the normal way the validator is used by other libs (kxser, kxcoro, ...) so new contributors that worked with it before will not have to use it differently than they are used to.

Reply:
I update the PR description to better state the reason for this change.